listenbrainz/webserver/templates/emails/id_paused.txt

```diff
@@ -0,0 +1,7 @@
+Hi {{ username }}!
+
+We have paused your listenbrainz ID due to sus activities, you wont be able to import new listens.
```

Comment:
sus, lol. :)

Lets go with:

"We have paused your ListenBrainz account ([user name]) because of a problem with your account. Our community manager will reach out to you with the details as to why we took this step. Feel free to [Contact Us] if you have any questions about this."

and 

"Your ListenBrainz account ([account name]) has been restore and is now accepting listens again."